### PR TITLE
Add Gala2019 page

### PR DESCRIPTION
**Pull request by samdsmx**

### CODE REVIEW

**Comment by lhchavez** [comment]
defer a @rcxr.

File: eventos/omegaup-gala-2019/index.html [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+        </div>
+    </nav> 
+</section>
+
+<section class="engine"><a href="https://mobiri.se/c">website builder</a></section><section class="header12 cid-rAuVZ1PT9q mbr-fullscreen mbr-parallax-background" id="header12-0">
+    <div class="container">
+            <div class="media-container">
+                <div class="col-md-12 align-center">
+                    <h1 class="mbr-section-title pb-3 mbr-white mbr-bold mbr-fonts-style display-1">Gala 2019 "Coding Latam"<br></h1>
+                    <p class="mbr-text pb-3 mbr-white mbr-fonts-style display-5"><strong>6:30 pm Saturday, October 19</strong><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```
Comment (rcxr) [inline eventos/omegaup-gala-2019/index.html]: Saturday, October 19, 6:30 PM
1) time is more specific and often goes afterwards
2) PM is preferred over pm and p.m. (see https://brians.wsu.edu/2016/05/16/am-pm/#targetText=It%20is%20now%20rare%20to,to%20get%20you%20into%20trouble.)

File: eventos/omegaup-gala-2019/index.html [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+    
+</section>
+
+<section class="countdown1 cid-rAv9oUpjn4" id="countdown1-d">
+    
+    
+    
+    <div class="container ">
+        
+        <h3 class="mbr-section-subtitle align-center mbr-fonts-style display-5">Join us on Saturday, October 19, 2019 &nbsp;to celebrate the 2nd Annual Gala of omegaUp "Coding Latam", our main fundraising event.&nbsp;We support and help prosper future Engineers in Latin America and you can be part of it!</h3>
```
Comment (rcxr) [inline eventos/omegaup-gala-2019/index.html]: Double space after "Saturday, October 19, 2019"

File: eventos/omegaup-gala-2019/index.html [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+            <div class="row timeline-element  separline">
+                <span class="iconsBackground">
+                    <span class="mbr-iconfont mbri-edit"></span>
+                </span>
+                <div class="col-xs-12 col-md-6 align-left ">
+                    <div class="timeline-text-content">
+                        <h4 class="mbr-timeline-title pb-3 mbr-fonts-style display-5">
+                            2017</h4>
+                        <p class="mbr-timeline-text mbr-fonts-style display-7">
+                            omegaUp is a 501(c)(3) non-profit with the mission of building the talent pipeline of Software Engineering in America Latina.<br><br>We started hosting the Peruvian Olympiad in Informatics<br>
```
Comment (rcxr) [inline eventos/omegaup-gala-2019/index.html]: Use Latin America to be consistent with the wording above.

File: eventos/omegaup-gala-2019/index.html [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+            <div class="row timeline-element reverse">
+                <span class="iconsBackground">
+                    <span class="mbr-iconfont mbri-growing-chart"></span>
+                </span>
+                <div class="col-xs-12 col-md-6 align-left">
+                    <div class="timeline-text-content">
+                        <h4 class="mbr-timeline-title pb-3 mbr-fonts-style display-5">
+                            2018
+                        </h4>
+                        <p class="mbr-timeline-text mbr-fonts-style display-7">Running diferent programs like <em>Mentors</em> and <em>Career Experiences</em> and <em>Community Contests</em> omegaUp grow up to:<br><br>488,654 code solutions&nbsp;<br>9,846 active users
```
Comment (rcxr) [inline eventos/omegaup-gala-2019/index.html]: grew up? en pasado?

**Comment by rcxr** [comment]
This looks good :)

**Pull request merged**